www/includes/easyparliament/templates/html/search/by-person.php

```diff
@@ -0,0 +1,108 @@
+                        <?php if ( $house == 4 ) { ?>MSPs only<?php } else { ?><a href="<?= $this_url->generate('html', array('house'=>4)) ?>">MSPs only</a><?php } ?> |
+                        <?php if ( $house == 3 ) { ?>MLAs only<?php } else { ?><a href="<?= $this_url->generate('html', array('house'=>3)) ?>">MLAs only</a><?php } ?></li>
+                    <li><a href="<?= $ungrouped_url->generate() ?>">Ungroup results</a></li>
+                </ul>
+
+                <p class="search-results-legend">The <?= isset($limit_reached) ? '5000 ' : '' ?>most recent mentions of the exact phrase <em class="current-search-term"><?= $searchstring ?></em>, grouped by speaker name.</p>
+                <?php } ?>
+
+                <table class="search-results-grouped">
+                    <thead>
+                        <tr>
+                            <th>Occurences</th>
+                            <th>Speaker</th>
+                            <th>Date range</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                    <?php foreach ( $speakers as $pid => $speaker ) { ?>
+                        <?php if ( $wtt && $pid == 0 ) { continue; } // skip heading count for WTT lords list ?>
+                        <tr>
+                            <td><?= $speaker['count'] ?></td>
+                            <td>
+                                <?php if ( $pid ) { ?>
+                                <a href="/search/?s=<?= _htmlentities( $searchstring ) ?>&amp;pid=<?= $pid ?><?= isset($wtt) ? '&amp;wtt=2' : '' ?>"><?= isset($speaker['name']) ? $speaker['name'] : 'N/A' ?></a>
+                                <?php if ( $house != 2 ) { ?>
+                                <span class="search-results-grouped__speaker-party">(<?= $speaker['party'] ?>)</span>
+                                <?= isset($speaker['office']) ? ' - ' . join('; ', $speaker['office']) : '' ?>
+                                <?php } ?>
+                                <?php } else { ?>
+                                <?= $speaker['name'] ?>
+                                <?php } ?>
+                            </td>
+                            <td>
+                            <?php if ( format_date($speaker['pmindate'], 'M Y') == format_date($speaker['pmaxdate'], 'M Y') ) { ?>
+                            <?= format_date($speaker['pmindate'], 'M Y') ?>
+                            <?php } else { ?>
+                            <?= format_date($speaker['pmindate'], 'M Y') ?>&nbsp;&ndash;&nbsp;<?= format_date($speaker['pmaxdate'], 'M Y') ?>
+                            <?php } ?>
+                            </td>
+                        </tr>
+                    <?php } ?>
+                    </tbody>
+                </table>
+            <?php } ?>
+            </div>
+
+            <?php include 'sidebar.php' ?>
+        </div>
+
+    </div>
+</div>
+
+<script type="text/javascript">
```

Comment:
Can the two JSs be combined, in form.php, or is there a reason they're separate?